framework/doc/content/source/bcs/VectorDivPenaltyDirichletBC.md

```diff
@@ -0,0 +1,26 @@
+# VectorDivPenaltyDirichletBC
```

Comment:
I think it would be helpful to link to the documentation for `VectorPenaltyDirichletBC` and highlight the difference between the two objects and when each should be used over the other. I can imagine a user being confused on this point

Reply:
yep!